src/contents/all/components/knot/Knots/Knots.tsx

```diff
@@ -34,6 +34,13 @@ const Knots = ({
 
   const hasKnots = knots && knots.length > 0;
 
+  if (hasKnots) {
+    knots = knots.sort(function(a,b){
```

Comment:
hmm not sure we want to reassign knots since it's coming in from props. Wondering if it might have weird issues when updating props after the initial sort. 

Thinking we can just handle this Rails side and add an order by postedAt to KnotsController#index